src/components/store-deploy-block.tsx

```diff
@@ -91,7 +96,7 @@ export const StoreDeployBlock: FunctionComponent<DocumentStoreAddressProps> = ({
           />
         </div>
         <div className="sm:flex pt-5">
-          <SecondaryButton onClick={() => setShowModal(false)} className="w-full mr-5 text-sm font-medium">
+          <SecondaryButton onClick={() => clearDeployStatus()} className="w-full mr-5 text-sm font-medium">
```

Comment:
as a user, it's not clear what cancel does + the message is not really helpful (you can leave but I started to deploy so I can cancel to leave?)

please review this

Reply:
Okay, I have updated the button text from cancel to close. It is more associate that the button will close the popup